Firestore/core/src/firebase/firestore/remote/connectivity_monitor_apple.mm

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2018 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "Firestore/core/src/firebase/firestore/remote/connectivity_monitor.h"
+
+#import <Foundation/Foundation.h>
```

Comment:
Do we actually use anything from Foundation in here? If you could strike this import this could become a straight .cc file. (One downside of that would that you'd need to `CFRelease` the `reachability_` reference.

Also unlike `free(3)`, `CFRelease` does not tolerate null inputs.